### PR TITLE
Updates integrity verification docs in Core

### PR DESCRIPTION
**Pull request by zvr**
This simply updates the definition of `Hash`, `IntegrityMethod`, and `verifiedUsing` in Core to mention other possible verification methods in profiles.

### CODE REVIEW

**Review by goneall — CHANGES_REQUESTED** [review]
I think we need more description on how this would apply to package verification.

If the goal is to provide the reader of the spec with guidance on how to verify a package, just stating that additional methods may be available will not be sufficient.  PR #602 includes enough specifics but my be too constrained.  I would suggest modifying PR #602 to generalize enough to resolve any concerns about over specifying the algorithm.

File: .github/workflows/validate-pr.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+name: validate PR
```
[the remark this answers is not in the document]
Comment (zvr) [reply .github/workflows/validate-pr.yml:1]: Of course not. Sorry, @goneall

**Comment by maxhbr** [comment]
@zvr This PR is conflicting

**Comment by zvr** [comment]
@maxhbr yes, because of my error of including the workflow file.

Unfortunately I  won't be able to correct it myself for a while...

**Comment by kestewart** [comment]
Moving this to 3.1 - since unlikely to be a breaking change (extending vocabulary options).

**Comment by zvr** [comment]
@kestewart conflicts fixed.

The workflow file is the same as the one present making it a no-op (I don't know how to actually remove it).

As explained in the tech call, this simply adds a sentence to three descriptions, that profiles may provide additional integrity methods.

**Review by kestewart — APPROVED** [review]

**Comment by kestewart** [comment]
@goneall - can you see if Alexios' updates have fixed the issues you flagged?   If it's good from your perspective, we should probably include it.    If not,  please move it back to 3.1 (I moved it to 3.0, so it would be considered).

**Review by goneall — APPROVED** [review]
Agree with Kate that with the additional work in verification, this documentation is now sufficient.

**Pull request merged**